q2lint.py

```diff
@@ -35,11 +35,18 @@ def main():
                           pathlib.Path('.').glob('**/*/package.json'))
     if npm_packages:
         import subprocess
+        import os
 
         for package in npm_packages:
             pkg_path = package.parent
-            cmd = 'cd %s && npm i && npm run build -- --bail && (git diff ' \
-                  '--quiet */bundle.js || bash -c "exit 42")' % pkg_path
+            install = 'cd %s && npm i --silent' % pkg_path
+            cmd = subprocess.run(install, shell=True,
+                                 stdout=open(os.devnull, 'wb'))
+            if cmd.returncode != 0:
+                errors.append('npm install failed: %s' % cmd.stderr)
```

Comment:
You can drop the `cmd.stderr` here, since it will always be empty (because `--silent` stops emitting to `stderr`)? I think just a warning that the install failed is sufficient, because the `build` step below will give us the information we need to solve the install/build issues.